src/openforms/js/components/form/edit/tabs.js

```diff
@@ -46,6 +46,14 @@ const SENSITIVE_BASIC = {
     }),
 };
 
+const SENSITIVE_READ_ONLY = {
```

Comment:
all these different configurations are getting out of hand, I think it's starting to make more sense to just compose them in the actual form component because the re-use potential diminishes with every configuration change.

We're defining groups of config options and then in components filtering out certain options, in other places we add options... it's completely unclear what options are now actually included from looking at the code.

No need to change it for this PR, but keep it in mind for the next change we need to make and grab the opportunity to clean this up.